examples/ecs/component_change_detection.rs

```diff
@@ -43,15 +43,15 @@ fn change_detection(query: Query<(Entity, &MyComponent), Changed<MyComponent>>)
     }
 }
 
-// By looking at trackers, the query is not filtered but the information is available
-fn tracker_monitoring(
-    query: Query<(
-        Entity,
-        Option<&MyComponent>,
-        Option<ChangeTrackers<MyComponent>>,
-    )>,
-) {
-    for (entity, component, trackers) in &query {
-        info!("{:?}: {:?} -> {:?}", entity, component, trackers);
+// By using `Ref`, the query is not filtered but the information is available
```

Comment:
This comment seems incorrect. I'm not sure that it was ever correct however.

Reply:
I think the original intended meaning is that the query is not filtered based on change detection, but you can still access the change detection info. I tried to preserve that meaning when rewriting this section of code.